code/modules/mob/living/carbon/xenomorph/xeno_defines.dm

```diff
@@ -1181,6 +1184,7 @@
 	destruction_allowed = XENO_NOBODY
 	dynamic_evolution = FALSE
 	allow_no_queen_actions = TRUE
+	allow_no_queen_evo = TRUE
```

Comment:
why are all these being added when this isn't an override

Reply:
![2023-07-27_16-42-20](https://github.com/cmss13-devs/cmss13/assets/115417687/87b58540-20c3-4568-ab5a-02e320017bb8)
It is?

Reply:
> ![2023-07-27_16-42-20](https://user-images.githubusercontent.com/115417687/256555441-87b58540-20c3-4568-ab5a-02e320017bb8.png)
> It is?

lmao my bad i got mixed up with the var you changed the comment of